### DISCUSSION TITLE
Выбираем наименование для рабочего репозитория концепции связности

### DISCUSSION THREAD

**Discussion by audetv** [audetv/cohesion#6] [Polls]
https://github.com/audetv/fct-search/issues/68#issuecomment-1583171740

В именование репозитория «of» как-то не хочется, хотя почему бы и нет.
concept of cohaesus - вообще в так это произносится?
https://github.com/audetv/concept-of-cohaesus
https://github.com/audetv/cohaesus 
https://github.com/audetv/cohaesus-concept
https://github.com/audetv/concept-cohaesus

Или напишите свой вариант в комментарии.

**Comment by iprst** [comment]
Можно не латинское, а английское написание cohesion (вместо cohaesus)
Хотя таким образом _есус_ символично превращается в _зион_.

Я за простоту тем не менее в любом случае!

**Reply by audetv** [reply]
https://github.com/audetv/cohesion

**Reply by iprst** [reply]
Как туда можно перенести добавленные файлы отсюда? Например целиком [папку](https://github.com/audetv/fct-search/tree/main/STT_ASR_zaznobin).

**Reply by iprst** [reply]
Или проще удалить здесь и загрузить там?

**Reply by iprst** [reply]
Можно ли вообще удалить здесь без того, чтобы сервер не пересобирался по-новой?

**Reply by audetv** [reply]
сейчас удалить без пересборки нельзя, но сейчас надо удалить. пусть сделает сборку, мне надо проверить токен. Все по плану

**Reply by iprst** [reply]
Тогда лучше удалите сами всю папку, когда будет правильно. Я просто загружу в новый репозиторий всё ещё раз.
